yarn.lock

```diff
@@ -25147,6 +25149,22 @@ winston@^3.2.1:
     triple-beam "^1.3.0"
     winston-transport "^4.4.2"
 
+winston@^3.6.0:
```

Comment:
the rest of the project is at `^3.2.1` - changing your deps to that should clear up the yarn.lock